nipype/interfaces/fsl/tests/test_auto_ApplyXfm.py

```diff
@@ -1,9 +1,9 @@
 # AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
 from __future__ import unicode_literals
-from ..preprocess import ApplyXfm
+from ..preprocess import ApplyXFM
```

Comment:
`nipype/interfaces/fsl/tests/test_auto_ApplyXFM.py` already exists. Should this file just be removed?